documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -777,15 +777,22 @@ spec:
       name: <storage_map> <7>
       namespace: <namespace>
   preserveStaticIPs: <8>
+  networkNameTemplate: <network_interface_template> <9>
+  pvcNameTemplate: <pvc_name_template> <10>
   targetNamespace: <target_namespace>
-  vms: <9>
-    - id: <source_vm> <10>
+  volumeNameTemplate: <volume_name_template> <11>
+  vms: <12>
+    - id: <source_vm> <13>
     - name: <source_vm>
```

Comment:
we agreed to do `source_vm1` and `source_vm2` , am i missing something ?

Reply:
This change covers https://issues.redhat.com/browse/MTV-1430.